library/std/src/diagnostics.qs

```diff
@@ -69,11 +69,37 @@ namespace Microsoft.Quantum.Diagnostics {
         body intrinsic;
     }
 
+    /// # Summary
+    /// Checks whether a qubit is in the |0⟩ or ground state, returning true if it is.
```

Comment:
"|0⟩ or ground state" gives an impression that those are different states. May be use different wording if possible? Like "AKA ground state" or something?

Reply:
Removed, per Mariia's suggestion